### DISCUSSION TITLE
Documentation as PDF?

### DISCUSSION THREAD

**Discussion by JunglePredator** [johnkerl/miller#848] [General]
Hi, could you make a PDF of the documentation for offline viewing?

I find it a lot easier to read that way.

**Comment by johnkerl** [comment]
Hi @JunglePredator -- at https://docs.readthedocs.io/en/stable/features.html it looks like readthedocs supports PDF generation only for Sphinx docs, and (unfortunately) I moved away from Sphinx and am much, much, much happier with mkdocs+material.

This appears to be a limitation of readthedocs. :(

**Comment by aborruso** [comment]
@johnkerl I publish my mkdocs site in github, using [this action](https://github.com/aborruso/arigadicomando/blob/master/.github/workflows/ci.yml). The static website is in [gh-pages branch](https://github.com/aborruso/arigadicomando/tree/gh-pages).

It's a static website and it's perfect for off-line use. For me, much more convenient and useful than a PDF

**Comment by JunglePredator** [comment]
PDF is better for me as I can load it on my e-reader is all.
